pkg/render/node.go

```diff
@@ -245,6 +245,19 @@ func (c *nodeComponent) Ready() bool {
 	return true
 }
 
+// CalicoSystemFinalizedObjects returns a list of objects that use the CNIFinalizer that should be
+// removed only after the CNI plugin is removed.
+func CalicoSystemFinalizedObjects() []client.Object {
```

Comment:
In light of the conversation we had outside of this PR (about generalising this to anything other than the CNI plugins resources), this should probably be named something specific to the CNI plugin's resources.

Reply:
I was just in the process of doing that :laughing: